Common/Brokerages/DefaultBrokerageModel.cs

```diff
@@ -258,5 +257,16 @@ public virtual ISettlementModel GetSettlementModel(Security security, AccountTyp
             return new ImmediateSettlementModel();
         }
 
+        /// <summary>
+        /// Gets a new buying power model for the security, returning the default model with the security's configured leverage.
+        /// For cash accounts, leverage = 1 is used.
+        /// </summary>
+        /// <param name="security">The security to get a buying power model for</param>
+        /// <param name="accountType">The account type</param>
+        /// <returns>The buying power model for this brokerage/security</returns>
+        public virtual IBuyingPowerModel GetBuyingPowerModel(Security security, AccountType accountType)
+        {
+            return security.BuyingPowerModel;
```

Comment:
I believe all of the other `GetXXXModel` methods return a new instance. Any particular reason why this one doesn't?

Reply:
I just did it to avoid the `switch` statement on the security type, only to return the default models already declared in the constructors.
I'll add the switch for consistency with the other methods in the class.